### PR TITLE
Make $outfit and $tattoo a type in ASH

### PR DESCRIPTION
**Pull request by gausie**
This is an breaking change for ASH. 

```
get_outfits() is now $outfits[]
all_normal_outfits() is now $outfits[]
outfit_treats(string) is now $outfit[].treats
outfit_tattoo(string) is now $outfit[].tattoo
```

Still in draft and we might not want to do it, but it addresses a long standing deficiency (in my opinion) in the constants available to our scripting runtimes.

This still allows custom outfits to be referenced with a string.

In the future we can add functions to check if you have a specific tattoo unlocked and if you current have it on your profile

### CODE REVIEW

File: src/data/tattoos.txt [hunk 1]
```diff
@@ -0,0 +1,306 @@
+1
```
Comment (jaadams5) [inline src/data/tattoos.txt:1]: New file, versioned according to KoLmafia idiom, and tab delimited?  Can't really verify the latter.

**Comment by gausie** [comment]
@jaadams5 I meant to make this a draft but still. Yes! We should talk about it here and give some warnings. These are the breaking changes:

```
get_outfits() is now $outfits[]
all_normal_outfits() is now $outfits[]
outfit_treats(string) is now $outfit[].treats
outfit_tattoo(string) is now $outfit[].tattoo
```

**Comment by heeheehee-kolmafia** [comment]
The breaking change that worries me the most is that all variables called "outfit" or "tattoo" will now cause problems with the Parser.

    // This won't generate an error...
    string outfit = "";
    // nor this
    if (outfit == "test") { }
    // but this will
    outfit = "test";

Typically this is made more explicit by marking new datatypes as reserved words.

**Comment by gausie** [comment]
Going to close this for now, branch will still exist though

**Pull request closed without merging**